musicmirror/src/youtube.js

```diff
@@ -1,4 +1,5 @@
-import axios from 'axios';
+//import axios from 'axios';
+axios = require('axios');
```

Comment:
This creates a compilation error as it is currently. Just add "const" and it'll be good to go.
`const axios = require('axios');`